client/components/Header/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Link } from 'react-router-dom';
+import './style.scss';
```

Comment:
I suggest using [CSS modules](https://github.com/webpack-contrib/css-loader). The syntax will be a bit different but we get the benefit of scoping css names.